cmd/utils.go

```diff
@@ -160,6 +161,24 @@ var globalProfiler interface {
 	Stop()
 }
 
+// Prints a given message after a requested delay.
+func delayPrint(message string, delay time.Duration) chan bool {
```

Comment:
> This message is only useful in the rare case of another process migrating the backend and your lock will block. Hence it is worth a little complexity in the code to print it only when it is needed and not print all the time.

@balamurugana you didn't read the message properly, the delayed print is added to avoid printing unless its necessary. Say if the lock was acquired with in a second which is most common.  Its not really about what the message string should look like.